model/event/subscribers.go

```diff
@@ -84,33 +89,18 @@ func (s *Subscriber) SetBSON(raw mgobson.Raw) error {
 }
 
 func (s *Subscriber) String() string {
-	subscriberStr := "NIL_SUBSCRIBER"
+	if stringer, ok := s.Target.(fmt.Stringer); ok {
+		return fmt.Sprintf("%s-%s", s.Type, stringer)
+	}
```

Comment:
it's my understanding that you can put this in as one of the cases in the type switch.